model/src/main/java/org/cbioportal/genome_nexus/model/my_variant_info_model/AlleleFrequency.java

```diff
@@ -0,0 +1,134 @@
+package org.cbioportal.genome_nexus.model.my_variant_info_model;
+
+import org.springframework.data.mongodb.core.mapping.Field;
+
+public class AlleleFrequency
+{
+
+    // allele frequency in total
+    @Field(value = "af")
+    private Float af;
```

Comment:
I think we should use `Double` instead of `Float` to be consistent with the other models.